packages/login-form/src/__snapshots__/LoginForm.test.tsx.snap

```diff
@@ -1,25 +1,34 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`LoginForm should match snapshot and styles 1`] = `
-.c0 {
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-flex-direction: column;
-  -ms-flex-direction: column;
-  flex-direction: column;
-  margin: 10px;
-}
-
-.c1 {
-  text-align: right;
-}
-
-<Component>
-  <styled.div>
+Object {
+  "asFragment": [Function],
```

Comment:
Snapshot looks weird, please use `asFragment` or `container` for checking  snapshots

Reply:
updated